samples/02_commands_framework/Modules/PublicModule.cs

```diff
@@ -60,6 +60,7 @@ public Task EchoAsync([Remainder] string text)
         public Task ListAsync(params string[] objects)
             => ReplyAsync("You listed: " + string.Join("; ", objects));
 
+        // Setting a custom ErrorMessage property will help clarify the precondition error
```

Comment:
This comment could be moved one line down so that it's more obvious what it's referring to.